test/e2e/kubectl.go

```diff
@@ -190,6 +195,101 @@ var _ = Describe("Kubectl client", func() {
 			}
 		})
 
+		It("support exec through an HTTP proxy", func() {
+			SkipIfProviderIs("local")
```

Comment:
Add a comment that we're skipping local b/c we only want to test an apiserver using https, and the local setup is just plain http